src/character.h

```diff
@@ -35,6 +35,7 @@
 #include "item_location.h"
 #include "magic_enchantment.h"
 #include "memory_fast.h"
+#include "move_mode.h"
```

Comment:
Please forward-declare steed_type instead, this header is huge and included in a lot of places, and it significantly increases compile time.